src/model/engines/pep8.nit

```diff
@@ -0,0 +1,140 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Pep/8 terminal engine for submissions
+#
+# TODO: Pep8Term is crap software to install and to execute.
+# Currently, this program assume that it is installed in the subdirectory `pep8term`.
+# Use `make pep8term` to install and compile it.
+module pep8
+
+import engine_base
+
+# The absolute path of a `file` in pep8term.
+# Aborts if not found
+fun pep8term(file: String): String do
+	var dir = "pep8term"
+	if not dir.file_exists then
+		print_error "{dir}: does not exists. Please install it (make pep8term)."
+		exit 1
+	end
+	file = dir.realpath / file
+	if not file.file_exists then
+		print_error "{file}: does not exists. Please check the pep8term installation."
+		exit 1
```

Comment:
Since this is used by the server, can we avoid `exit` and `abort` so we don't kill the server if a user makes an error?

I understand this is lib so... maybe use the following solution:

EngineBase provides the service `check_engine` that return true or false depending if the engine can be used or not.
It is the responsability of the client (the web server) to check if the engine is usable before calling the process. So you can still have your abort in the code. But the abort should occur only if someone forgot to call the `check_engine` before